Makefile

```diff
@@ -3,13 +3,19 @@
 
 BASE_IMG_URL := http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2016-03-18/2016-03-18-raspbian-jessie-lite.zip
 BASE_IMG_NAME := $(basename $(notdir $(BASE_IMG_URL))).img
+SPRINGBOARD := root/usr/share/tingbot/springboard.tingapp
 
-build/tingbot-os.deb: $(shell find root)
+build/tingbot-os.deb: $(SPRINGBOARD) $(shell find root)
```

Comment:
`shell find root` returns all the files under `root/`, so no need to explicitly list it as a dep here.